Lib/idlelib/ParenClose.py

```diff
@@ -0,0 +1,115 @@
+"""
+Parens and Ticks Closing Extension
+
+When you hit left paren or tick,
+automatically creates the closing paren or tick.
+
+Author: Charles M. Wohlganger
+        charles.wohlganger@gmail.com
+
+Last Updated: 13-Aug-2017 by Charles Wohlganger
+
+Add to the end of config-extensions.def :
```

Comment:
Config options will be added to config-general.def.  The general tab already needs to be split for another issue.  This issue, and a complete patch, will depend on how we do that, and then how to lay out the option buttons.